src/Protocol/Protocol_1_9.cpp

```diff
@@ -3267,6 +3267,23 @@ eBlockFace cProtocol_1_9_0::FaceIntToBlockFace(Int32 a_BlockFace)
 
 
 
+eHand cProtocol_1_9_0::HandIntToEnum(Int32 a_Hand)
+{
+	// Convert hand parameter into eHand enum
+	switch (a_Hand)
+	{
+	case MAIN_HAND: return eHand::hMain;
+	case OFF_HAND: return eHand::hOff;
+	default:
```

Comment:
The case statements should be indented as well.

Reply:
Done.